packages/botonic-plugin-flow-builder/tests/bot-action.test.ts

```diff
@@ -0,0 +1,83 @@
+import { INPUT } from '@botonic/core'
+import { describe, test } from '@jest/globals'
+
+import { BOT_ACTION_PAYLOAD_SEPARATOR } from '../src/constants'
+import { FlowBuilderAction, FlowText } from '../src/index'
+import { ProcessEnvNodeEnvs } from '../src/types'
+import { testFlow } from './helpers/flows'
+import {
+  createFlowBuilderPlugin,
+  createRequest,
+  getActionRequest,
+} from './helpers/utils'
+
+describe('Check the contents returned by the plugin', () => {
+  process.env.NODE_ENV = ProcessEnvNodeEnvs.PRODUCTION
+  const flowBuilderPlugin = createFlowBuilderPlugin(testFlow)
+
+  test('The user does the first interaction', async () => {
+    const request = createRequest({
+      input: { data: 'Hola', type: INPUT.TEXT },
+      isFirstInteraction: true,
+      plugins: {
+        // @ts-ignore
+        flowBuilderPlugin,
+      },
+    })
+    await flowBuilderPlugin.pre(request)
+    const actionRequest = getActionRequest(request)
+    const { contents } = await FlowBuilderAction.botonicInit(actionRequest)
```

Comment:
Observation: As @AinaVendrell said with another part, maybe this 3 lines can be encapsulated also within a function because you will have to execute them in every test